locator/index.html

```diff
@@ -50,7 +50,8 @@
     }
 
  </style>
-
+<div class="_locator">
```

Comment:
Hey the branch looks great! I have just two small suggestions:
Can you please don't give class name with simbols or number at the beginning? :) Good practice 

I was wondering if you really need this 'locator' div since it has no style rule attach. Can't you place there just the h2? Or it doesn't work without?